client-device-connection/src/test/java/org/eclipse/hono/deviceconnection/common/CacheBasedDeviceConnectionInfoTest.java

```diff
@@ -396,7 +397,7 @@ public void testGetCommandHandlingAdapterInstancesWithExpiredEntry(final VertxTe
     @Test
     public void testGetCommandHandlingAdapterInstancesWithTerminatedAdapterInstance(final VertxTestContext ctx) {
 
-        final AdapterInstanceStatusProvider statusProvider = mock(AdapterInstanceStatusProvider.class);
+        final AdapterInstanceStatusProvider statusProvider = Mockito.mock(AdapterInstanceStatusProvider.class);
```

Comment:
for consistency, can you please revert this? We usually statically import the Mockito methods ...

Reply:
Sorry about all the formatting changes. I hadn't noticed them myself even but will definitely clean them up. Should have done that before asking you to look this over.

Reply:
no worries :-)